lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -463,7 +463,7 @@ private function parseColumnCollationFromSQL($column, $sql)
     private function parseColumnCommentFromSQL($column, $sql)
     {
         if (preg_match(
-            '{[\s(,](?:'.preg_quote($this->_platform->quoteSingleIdentifier($column)).'|'.preg_quote($column).')
+            '{[\s(,](?:\W'.preg_quote($this->_platform->quoteSingleIdentifier($column)).'\W|\W'.preg_quote($column).'\W)
```

Comment:
As pointed out on https://github.com/doctrine/dbal/pull/2865#issuecomment-341418300 this regex doesn't work for some scenarios, can you please check that?

Reply:
I have checked the solution but the problem still occurs. You could add the following test case to reproduce the error:

```php
'Column with numeric but no comment' => [
	false, 'a', 'CREATE TABLE "a" ("a" NUMERIC(10, 0) NOT NULL, "b" CLOB NOT NULL --(DC2Type:array)' . "\n" . ', "c" CHAR(36) NOT NULL --(DC2Type:guid)' . "\n" . ')',
],
```

In this case I would assume `false` for column "a" but it returns `(DC2Type:guid)`